routes/caniuse/caniuse.ts

```diff
@@ -0,0 +1,23 @@
+import { Request, Response } from "express";
+
+import { seconds } from "../../utils/misc.js";
+
+import { createResponseBodyHTML, sanitizeBrowsersList } from "./lib/index.js";
+
+type Params = { feature: string };
+type Query = { browsers?: string };
+type IRequest = Request<Params, any, any, Query>;
+
+export default async function route(req: IRequest, res: Response) {
+  const feature = req.params.feature;
+  const browsers = sanitizeBrowsersList(req.query.browsers?.split(/,/g) || []);
```

Comment:
Q: what's the difference between `/,/g` and just `.split(",")`?

Reply:
Hmm I was probably planning to use `/,\s*/`, but let's stick with `","` to keep it simple (and hope people form the request right).